client_app/src/components/shelter/MoreDetailsBox.js

```diff
@@ -1,28 +1,25 @@
 import React from "react";
 import Modal from 'react-modal';
-const MoreDetailsBox = () => {
 
-  const [isMOpen, setIsMOpen] = React.useState(false);
+export const MoreDetailsBox = props => {
+  const {isMModalOpen, setIsMModalOpen, onSignUpVolunteersClick} = props;
 
   return (
     <div className="modal-overlay">
-      <button onClick = {() => setIsMOpen(true)}>Open Modal</button>
-      <Modal isOpen = {isMOpen} 
-        onRequestClose = {() => setIsMOpen(false)}
+      <Modal isOpen = {isMModalOpen} 
+        onRequestClose = {() => setIsMModalOpen(false)}
         ariaHideApp = {false} >
         <h2>Shift Details</h2>
         <p>Start Time: 7:30 am </p>
         <p>End Time: 9:30 am </p>
         <p>Requested Volunteers: 4 </p>
-        <p>Signed Up Volunteers: 1 </p>
+        <p><button onClick={onSignUpVolunteersClick}>^</button>Signed Up Volunteers: 1 </p>
         <p>Required Volunteers: 2 </p>
         <button >Request More Volunteers</button>
         <button> Sign-up of Volunteers</button>
```

Comment:
example you can use the general name: .button in the css file or you could add className = "signUpButton" and then the corresponding name of the component in the css file is .signUpButton, let me know if you need further clarification on this

Reply:
the class name goes inside the opening <button className = "signUpButton"> Sign-up of Volunteers</button>